File: CPP/Petshop.cpp
#include <iostream>
#include <string>

using namespace std;

class Petshop{
    private:
        string id;
        string nama;
        string kategori;
        int harga;
    
    public:
        Petshop(){//konstruktor
            this->id = "";
            this->nama = "";
            this->kategori = "";
            this->harga= 0;
        }

        Petshop(string id, string nama, string kategori, int harga){
            this->id = id;
            this->nama = nama;
            this->kategori = kategori;
            this->harga = harga;
        }

        //setter getter
        string get_id(){
            return this->id;
        }

        void set_id(string id){
            this->id = id;
        }

        string get_nama(){
            return this->nama;
        }

        void set_nama(string nama){
            this->nama = nama;
        }

        string get_kategori(){
            return this->kategori;
        }

        void set_kategori(string kategori){
            this->kategori = kategori;
        }

        int get_harga(){
            return this->harga;
        }

        void set_harga(int harga){
            this->harga = harga;
        }

        ~Petshop(){//destruktor

        }
};

File: PHP/Petshop.php
<!-- Saya Klara Ollivviera Augustine Gunawan dengan NIM 2306205 
mengerjakan soal Tugas Praktikum 1 dalam mata kuliah DPBO 
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin -->

<?php
class Petshop{
    private $id = "";
    private $nama = "";
    private $kategori = "";
    private $harga = 0;
    private $foto = "";

    public function __construct($id = "", $nama = "", $kategori = "", $harga = 0, $foto = ""){
        $this->id = $id;
        $this->nama = $nama;
        $this->kategori = $kategori;
        $this->harga = $harga;
        $this->foto = $foto;
    }

    public function setId($id){
        $this->id = $id;
    }

    public function getId(){
        return $this->id;
    }

    public function setNama($nama){
        $this->nama = $nama;
    }

    public function getNama(){
        return $this->nama;
    }
    public function setKategori($kategori){
        $this->kategori = $kategori;
    }

    public function getKategori(){
        return $this->kategori;
    }
    public function setHarga($harga){
        $this->harga = $harga;
    }

    public function getHarga(){
        return $this->harga;
    }

    public function setFoto($foto){
        $this->foto = $foto;
    }

    public function getFoto(){
        return $this->foto;
    }
}
?>